index.html

```diff
@@ -0,0 +1,200 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Dubai Airshow</title>
+</head>
+
+<body>
+  <header>
+    <img class="humburger" src="assets/img/hamburger-menu.svg" alt="Humburger">
+    <nav id="mobile-menu">
+      <ul class="mobile-menu-list">
+        <li class="mobile-menu-item">
+          <img id="cancel-btn" src="assets/img/cancelBtn.png" alt="Cancel">
+        </li>
+        <li class="mobile-menu-item">
+          <a href="about.html">About</a>
+        </li>
+        <li class="mobile-menu-item">
+          <a href="#program-section">Features</a>
+        </li>
+        <li class="mobile-menu-item">
+          <a href="#advisors-section">Advisors</a>
+        </li>
+        <li class="mobile-menu-item">
+          <a href="about.html#partners-section">Partners</a>
+        </li>
+        <li class="mobile-menu-item">
+          <a href="https://www.dubaiairshow.aero/news/industry-news" target="_blank" rel="noopener noreferrer">Media</a>
+        </li>
+      </ul>
+    </nav>
```

Comment:
- [ ] [OPTIONAL] Good job so far but I think you should adjust your top bar right position. Please make sure to make it similar to the original design 😄 

#### You design
![top](https://github.com/tajemouti/first-capstone/assets/6615394/a2646216-c5e8-4e82-9841-a79621826d06)

#### Original design
![Capture](https://github.com/tajemouti/first-capstone/assets/6615394/1fc95cd3-7896-4ce9-b04b-4b92e3b4a789)